packages/bot-web-ui/src/pages/bot-builder/quick-strategy/parts/loss-threshold-warning-dialog.scss

```diff
@@ -13,21 +13,23 @@
                 max-width: 90%;
             }
         }
+
         &__footer {
             @include mobile {
                 display: flex;
                 width: 100%;
-                flex-wrap: unset;
             }
         }
-        &__button:not(:last-child) {
+
+        &__button {
             @include mobile {
-                margin-right: 6px;
+                margin-left: 0;
```

Comment:
We should use `margin-inline-start` instead of margin-left @maryia-matskevich-deriv